GameData/SPEngine/Families/V.cfg

```diff
@@ -160,7 +192,7 @@ SPEFamily:NEEDS[RealFuels]
 			amount = 0.5
 		}
 		// Should really have a TEATEB IGNITOR_RESOURCE on all these MVacs, but then the LR91s would have to carry it too.  Let's not bother
-		maxIgnitions = 1
+		maxIgnitions = 4
```

Comment:
Trouble with this is that according to RO's configs the Merlin 1BVac only had a single ignition.
Since ignitions can't go backwards, I'd say make this one 2, same as the AJ-11-Kero before it.

Reply:
I guess I'll merge it then make this change myself.